site/content/docs/02-template-syntax.md

```diff
@@ -193,15 +193,19 @@ Content that is conditionally rendered can be wrapped in an if block.
 Additional conditions can be added with `{:else if expression}`, optionally ending in an `{:else}` clause.
 
 ```sv
-{#if porridge.temperature > 100}
-	<p>too hot!</p>
-{:else if 80 > porridge.temperature}
-	<p>too cold!</p>
-{:else}
-	<p>just right!</p>
-{/if}
+<p>
+	{#if porridge.temperature > 100}
+		too hot!
+	{:else if 80 > porridge.temperature}
+		too cold!
+	{:else}
+		just right!
+	{/if}
+</p>
 ```
 
+(Blocks don't have to wrap elements, they can also wrap text within elements!)
```

Comment:
I'm okay with this addition since it doesn't hurt to mention, but I'm against changing the tutorial or enhancing it with more info. I don't think that this is benefitial to someone new to Svelte to get confronted with this extra information at that time, especially if that someone already thought so intuitively.